### PR TITLE
Translated the about BeeWare project page to portuguese pt_BR

### PR DESCRIPTION
**Pull request by Rkanehisa**
Work in Progress

Translating the About BeeWare project page to brazillian protuguese.

### CODE REVIEW

**Comment by eliasdorneles** [comment]
Hi @Rkanehisa -- is this ready to be reviewed and merge?

**Comment by Rkanehisa** [comment]
Not yet, today I will finish it today.

**Comment by Rkanehisa** [comment]
It's ready now for review and merge.

File: content/project/about/contents+pt.lr [hunk 1]
```diff
@@ -0,0 +1,131 @@
+body:
+
+História
+=======
```
Comment (eliasdorneles) [inline content/project/about/contents+pt.lr]: I'm seeing some warnings due to the "title underline" thing not being aligned with the title text, for this and for other titles.
Could you fix those?

File: content/project/about/contents+pt.lr [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+Python tem se provado uma linguagem altamente capaz - de acessível para iniciantes, mas poderosa nas mãos de experts. O Python está rapidamente se tornando uma parte importante de  análises de dados, computaçãoi ciêntífica, e ambiente de desenvolvimento web. E iniciativas como "Django Girls" demonstraram que usando o Python, é possível fornecer cursos de treinamento de 1 dia que introduzam pessoas sem experiência de programação para o Python; e no final de um dia, os participantes desenvolveram e implantaram um site dinâmico com banco de dados, implantado em um servidor visível publicamente.
+
+.. _Django Girls: https://djangogirls.org/
+
+No entanto, a experiência do desenvolvedor para alguém que escreve Python às vezes reflete mais do início dos anos 90 do que o novo milênio. O ambiente de desenvolvimento padrão para Python ainda é uma janela de console 80x25. Para os usuários que vieram de um ambiente Windows ou Apple, onde eles são usados para interfaces gráficas altamente detalhadas e bem desenhadas, esse flashback de 30 anos pode ser uma experiência completamente alienanadora.
```
Comment (eliasdorneles) [inline content/project/about/contents+pt.lr]: A little typo here: `alienanadora`.

**Comment by Rkanehisa** [comment]
Fixed title underline too short and the typos.

I have a question. Should I create another branch if I am translating
another page or I should use the same branch I am working now?

Em 17 de outubro de 2017 21:59, Elias Dorneles <notifications@github.com>
escreveu:

**Comment by eliasdorneles** [comment]
Hey @Rkanehisa -- thanks for contributing !
I've just merged this, so it's good to create another branch.
But first, you'll want to [synchronize your fork](https://help.github.com/articles/syncing-a-fork/).

**Pull request merged**